lib/job.js

```diff
@@ -378,7 +381,7 @@ Job.prototype.finished = function(){
         //
```

Comment:
after merging I noticed that clearInterval could be called inside removeListeners to avoid one line of code :D